### PR TITLE
Make deployment easier

### PR DESCRIPTION
**Pull request by CodeSandwich**
Improve deployment script and its docs. It includes changes from branch `fixVerifyRegistry`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,168 @@
...
+
+# OPTIONAL
+# The file to write deployment addresses to. Default is `./deployment_<BLOCKCHAIN_NAME>.json`.
+export DEPLOYMENT_FILE="FILE"
+```
+
+Set up environment variables configuring the deployed contracts:
+
+```bash
+# OPTIONAL
```
Comment (manuelpolzhofer) [inline README.md:80]: 👍 nice for adding the explanations

File: README.md [hunk 2]
```diff
@@ -1,23 +1,168 @@
...
+scripts/deploy.sh
+```
+
+### Deploying to Polygon Mumbai
+
+Polygon Mumbai is supported by dapp.tools' `seth` in versions **newer than** 0.11.0.
+If no such version is officially released yet, you must install it from `master`:
+
+```bash
+git clone git@github.com:dapphub/dapptools.git
```
Comment (manuelpolzhofer) [inline README.md:150]: It is not required to clone the repo it is already in  `~/.dapp/dapptools` but maybe it is cleaner by extra cloning it.

File: scripts/deploy.sh [hunk 3]
```diff
@@ -16,11 +16,11 @@ addValuesToFile() {
...
 }
 
-DEPLOYMENT_FILE=${1:-./deployment_$(seth chain).json}
-GOVERNANCE=${1:-$ETH_FROM}
-DEFAULT_IPFS_IMG=${1:-QmcjdWo3oDYPGdLCdmEpGGpFsFKbfXwCLc5kdTJj9seuLx}
-[ -z "$CYCLE_SECS" ] && (( CYCLE_SECS = 7 * 24 * 60 * 60 )) # 1 week
-[ -z "$LOCK_SECS" ] && (( LOCK_SECS = 30 * 24 * 60 * 60 )) # 30 days
+DEPLOYMENT_FILE=${DEPLOYMENT_FILE:-./deployment_$(seth chain).json}
+GOVERNANCE=${GOVERNANCE:-$ETH_FROM}
+DEFAULT_IPFS_IMG=${DEFAULT_IPFS_IMG:-QmcjdWo3oDYPGdLCdmEpGGpFsFKbfXwCLc5kdTJj9seuLx}
```
Comment (manuelpolzhofer) [inline scripts/deploy.sh:21]: 👍

**Review by manuelpolzhofer — APPROVED** [review]
nice it makes the deploy script more stable and the `dapp verify-contract` up to the latest version.

File: README.md (hunk 2 above)
Comment (CodeSandwich) [reply README.md:150]: Hmm, on my Linux machine there is `~/.dapp`, but it doesn't contain `dapptools`, only the local testnet keystore :thinking:

**Pull request merged**